docs/src/main/asciidoc/_developer-guide/Jakarta_Persistence_Reference_Guide.adoc

```diff
@@ -698,33 +698,20 @@ Updated `jboss-deployment-structure.xml`:
 [[hibernate-properties]]
 == Hibernate properties
 
-WildFly automatically sets the following Hibernate properties (if
+WildFly automatically sets or checks the following Hibernate properties (if
 not already set in persistence unit definition):
 
 [cols=",",options="header"]
 |=======================================================================
 |Property |Purpose
 
-|hibernate.id.new_generator_mappings =true |New applications should let
-this default to true, older applications with existing data might need
-to set to false (see note below). It really depends on whether your
-application uses the @GeneratedValue(AUTO) which will generates new key
-values for newly created entities. The application can override this
-value (in the persistence.xml).
+|hibernate.id.new_generator_mappings can no longer be set to false by applications as per Hibernate 6 change to only support hibernate.id.new_generator_mappings=true.
```

Comment:
This is not a 2 cell table row any more.

Also, 
s/per Hibernate 6/per the Hibernate 6/g

Reply:
Also, 'can' should be 'Can' -- capitalize the first word in the 'Purpose' cell.